spline-shadow/client-ui/src/app/components/lineage/lineage-details/schema/schema.component.ts

```diff
@@ -34,33 +35,21 @@ export class SchemaComponent implements OnInit {
   @Input()
   schema: any
 
-  constructor() { }
+  property: any
```

Comment:
Is there more specific type for `property`?

Reply:
I didn't created the types for property as it might comes from the swagger generation. I will create if if necessary during the rest api integration.